<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>set_voltage_current_dlg</class>
 <widget class="QDialog" name="set_voltage_current_dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>一次二次额定值设置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>一次额定电压：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="primaryVoltageLineEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>单位：V</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>二次额定电压：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="secondaryVoltageLineEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>单位：V</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>一次额定电流：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="PrimaryCurrentLineEdit"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>单位：A</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>二次额定电流：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="secondaryCurrentLineEdit"/>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>单位：A</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OKPushButton">
       <property name="text">
        <string>确定(&amp;O)</string>
       </property>
       <property name="icon">
        <iconset resource="sim_iec61850.qrc">
         <normaloff>:/sim_iec61850/Resources/save.gif</normaloff>:/sim_iec61850/Resources/save.gif</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="canelPushButton">
       <property name="text">
        <string>取消(&amp;C)</string>
       </property>
       <property name="icon">
        <iconset resource="sim_iec61850.qrc">
         <normaloff>:/sim_iec61850/Resources/del.gif</normaloff>:/sim_iec61850/Resources/del.gif</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="sim_iec61850.qrc"/>
 </resources>
 <connections/>
</ui>
